app/server/models/credit_transfer.py

```diff
@@ -47,6 +46,13 @@ class CreditTransfer(ManyOrgBase, BlockchainTaskableBase):
 
     token_id        = db.Column(db.Integer, db.ForeignKey(Token.id))
 
+    # Present status, and time of last update (according to worker) to ensure the present blockchain_status 
+    # is the newest (since order of ack's is not guaranteed)
+    blockchain_status   = db.Column(db.Enum(BlockchainStatus), default=BlockchainStatus.PENDING)
+    blockchain_hash = db.Column(db.String)
+    last_worker_update  = db.Column(db.DateTime)
+    messages = db.relationship('WorkerMessages', backref='credit_transfer', lazy=True)
```

Comment:
given that exchanges also use the above logic, wouldn't it be better to add this to the blockchain-taskable base class?